app/src/androidTest/java/com/sdp/movemeet/Profile/HomeScreenActivityToUserProfileActivity.java

```diff
@@ -47,6 +49,9 @@
     @Rule
     public ActivityTestRule<HomeScreenActivity> mActivityTestRule = new ActivityTestRule<>(HomeScreenActivity.class);
 
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION);
+
     @Test
     public void homeScreenActivityToUserProfileActivity() {
         /*onView(withId(R.id.signInButton)).perform(click());
```

Comment:
If this test fails because of the login part, the best idea is to sign in programmatically using `FirebaseAuth` and launching the activity afterwards using `ActivityScenario.launch()` (this would mean not using the `ActivityTestRule`)

Reply:
I don't really looked into this test, since it was already commented when I merged it with my code, so I just assumed it didn't work. But I'll try to follow your advise, and see if it works now !